src/shared/extensionGlobals.ts

```diff
@@ -64,7 +66,10 @@ interface ToolkitGlobals {
     regionProvider: RegionProvider
     sdkClientBuilder: AWSClientBuilder
     telemetry: TelemetryService & { logger: TelemetryLogger }
-    templateRegistry: CloudFormationTemplateRegistry
+    templateRegistry: {
+        cfn: CloudFormationTemplateRegistry
+        buildspec: BuildSpecTemplateRegistry
```

Comment:
It's no longer a "templateRegistry", but some sort of "watch all the things" module. Can rename in a later PR.